docs/docs/recipes/working-with-images.md

```diff
@@ -78,50 +80,52 @@ This is an **escape route** for [specific use cases](/docs/static-folder/#when-t
 │       └── fiesta.jpg
 ```
 
-2. In `index.js`, add an `<img>` tag with the `src` as the relative path of the file from the `static` folder, and add an `alt` attribute [describing the image](https://webaim.org/techniques/alttext/):
+2. Dans `index.js`, ajoutez une balise `<img>` avec `src` comme chemin relatif du fichier depuis le dossier `static`, et ajoutez un attribut `alt` [décrivant l'image](https://webaim.org/techniques/alttext/):
 
 ```jsx:title=src/pages/index.js
 import React from "react"
 
-export default () => (
-  <img src={`fiesta.jpg`} alt="A dog smiling in a party hat" /> // highlight-line
-)
+export default function Home() {
```

Comment:
Idem, ne pas changer les fontions

Reply:
you asked me me to use the original function in the previous commit? are you sure i should translate them to french and then keep them

Reply:
Yes, it is code, JS still don't understand French yet ;)

This is what you said in my previous commit, could you be please more clear with the changes